src/main/java/com/webs/user/AppSecurityConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package com.webs.user;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Generic configuration for the application
```

Comment:
The class name seems pretty informative of what its purpose is, but the comment seems a little generic. Either it can be more specific or we can maybe drop the comment.